core/src/main/java/com/orange/ouds/core/component/OudsControlItem.kt

```diff
@@ -91,77 +96,79 @@ internal fun OudsControlItem(
+                            if (element?.enabled == false) padding(horizontal = controlItemTokens.spacePaddingInline.value) else this
+                        },
+                    horizontalArrangement = Arrangement.spacedBy(controlItemTokens.spaceColumnGap.value)
                 ) {
-                    Text(text = label, style = OudsTheme.typography.label.default.large, color = labelColor(state = state, error = error))
-                    if (!additionalLabel.isNullOrBlank()) {
-                        Text(
-                            text = additionalLabel,
-                            style = OudsTheme.typography.label.strong.medium,
-                            color = additionalLabelColor(state = state)
-                        )
-                    }
-                    if (!helperText.isNullOrBlank()) {
-                        Text(
-                            text = helperText,
-                            style = OudsTheme.typography.label.default.medium,
-                            color = helperTextColor(state = state)
-                        )
+                    leadingElement?.let { LeadingTrailingBox(leadingElement) }
+                    Column(
+                        modifier = Modifier
+                            .weight(1f)
+                            .align(Alignment.CenterVertically),
+                        verticalArrangement = Arrangement.spacedBy(controlItemTokens.spaceRowGap.value)
+                    ) {
+                        Text(text = label, style = OudsTheme.typography.label.default.large, color = labelColor(state = state, error = error))
+                        if (!additionalLabel.isNullOrBlank()) {
+                            Text(
+                                text = additionalLabel,
+                                style = OudsTheme.typography.label.strong.medium,
+                                color = additionalLabelColor(state = state)
+                            )
+                        }
+                        if (!helperText.isNullOrBlank()) {
+                            Text(
+                                text = helperText,
+                                style = OudsTheme.typography.label.default.medium,
+                                color = helperTextColor(state = state)
+                            )
+                        }
                     }
+                    trailingElement?.let { LeadingTrailingBox(trailingElement) }
+                }
+                if (divider) {
+                    OudsHorizontalDivider(color = dividerColor(state = state, error = error))
                 }
-                trailingElement?.let { LeadingTrailingBox(trailingElement) }
             }
-            if (divider) {
-                OudsHorizontalDivider()
+            if (error != null && error.description.isNotBlank()) {
+                ErrorDescriptionText(text = error.description)
```

Comment:
The error description here and in `OudsTextInput` is displayed as a text and is not limited to accessibility services now, so I think if we should modify the KDoc and remove `for the accessibility services`:

```
A mandatory localized description explaining the error for the accessibility services.
```

And maybe rename `description` to `message` (we chose `description` because it is almost identical to `contentDescription` and `message` is often used for objects related to errors).